XmlFormatter/src/DataContainer/VersionCompare.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Octokit;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace XmlFormatter.src.DataContainer
+{
+    class VersionCompare
+    {
+        private readonly bool gitHubIsNewer;
+        public bool GitHubIsNewer => gitHubIsNewer;
+
+        private readonly Version localVersion;
+        public Version LocalVersion => localVersion;
+
+        private readonly Version gitHubVersion;
+        public Version GitHubVersion => gitHubVersion;
+
+        private readonly Release lastestRelease;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private field 'lastestRelease'.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=4995221)